py/torch_tensorrt/_enums.py

```diff
@@ -24,9 +23,9 @@ class dtype(Enum):
     f32 = auto()
     f64 = auto()
     b = auto()
-    # TODO: Enable FP8 and BF16
-    # f8 = auto()
-    # bf16 = auto()
+
+    f8 = auto()
+    bf16 = auto()
```

Comment:
Make sure this is aligned with main, otherwise the enum values would change version to version